lua/class_definitions.lua

```diff
@@ -84,11 +84,6 @@ classes = {
             nv_cached = { type = "bool", writable = true },
         },
         functions = {
-            { name = "add_effect", rval = nil, args = { "string", "int" } },
```

Comment:
Can't find anything about effects in that mod. The mod changes the base stats directly.